src/Form/matching.js

```diff
@@ -822,10 +845,10 @@ const getRelatedText = (el, form, cssSelector) => {
     const label = scope.querySelector('label')
     if (label) {
         // Try searching for a label first
-        trimmedText = removeExcessWhitespace(getText(label))
+        trimmedText = getTextShallow(label)
     } else {
         // If the container has a select element, remove its contents to avoid noise
-        trimmedText = removeExcessWhitespace(extractElementStrings(scope).join(' '))
+        trimmedText = extractElementStrings(scope).join(' ')
```

Comment:
The calls to `removeExcessWhitespace` were redundant, because that's already done in the underlying methods `getShallowText` and `extractElementStrings`.